CONTRIBUTING.md

```diff
@@ -14,7 +14,7 @@ from your employer to make the contributions.
 
 We love pull requests. Here is a quick guide:
 
-1. You need to have a JDK (at least version 1.8) installed.
+1. You need to have a JDK (at least version 11) installed.
```

Comment:
Could we revert that and put mockito to 4.11.0 which still works with Java 8?

Reply:
Ok, tried to revert your changes but as CPDs API is very restricted with `final` classes and `package-private` constructors it is not easy to mock `Mark` for the adjusted test in `CpdReporterTest`.

Let's use Java 11 from now on as Mockito 11 can cope with `final` `Mark` class.